### PR TITLE
make sealer log more readable.

### PR DESCRIPTION
**Pull request by VinceCui**
add some phase logs;
silence log in sshcmd.go;
fix bug: delete will fail if kubeadm has been deleted;
fix bug: delete will fail if sealer-registry has been deleted;
fix bug: session not closed

### CODE REVIEW

**Comment by allencloud** [comment]
start to do someting
succeed in doing something
failed to do something

There is grammar error in `do something failed ` .

@VinceCui @fanux

**Comment by VinceCui** [comment]
> 

fixed

File: apply/applytype/local.go [hunk 1]
```diff
@@ -91,103 +92,135 @@ func (c *Applier) unMountClusterImage() error {
...
 		return err
 	}
-	if err = c.upgradeCluster(mj, nj); err != nil {
+
+	upgraded, err := c.upgradeCluster(mj, nj)
+	if err != nil {
 		return err
 	}
+
+	if !scaled && !upgraded {
```
Comment (fanux) [inline apply/applytype/local.go:129]: I think better to those log put into the scale and upgrade function.

File: apply/applytype/local.go [hunk 2]
```diff
@@ -91,103 +92,135 @@ func (c *Applier) unMountClusterImage() error {
 }
 
 func (c *Applier) changeCluster() error {
+	logger.Info("There is an existing cluster, start changing this cluster")
```
Comment (fanux) [inline apply/applytype/local.go:95]: `The cluster already exists, start to reconcile the cluster`

File: apply/applytype/local.go [hunk 3]
```diff
@@ -91,103 +92,135 @@ func (c *Applier) unMountClusterImage() error {
...
 }
 
-func (c *Applier) scaleCluster(mj, md, nj, nd []string) error {
+func (c *Applier) scaleCluster(mj, md, nj, nd []string) (bool, error) {
 	if len(mj) == 0 && len(md) == 0 && len(nj) == 0 && len(nd) == 0 {
-		return nil
+		return false, nil
 	}
+
+	logger.Info("Start scaling this cluster")
```
Comment (fanux) [inline apply/applytype/local.go:143]: `Start scaling the cluster`

File: apply/applytype/local.go [hunk 4]
```diff
@@ -91,103 +92,135 @@ func (c *Applier) unMountClusterImage() error {
...
 	}
 
 	err = applier.DoApply(c.ClusterDesired)
 	if err != nil {
-		return err
+		return true, err
 	}
-	return nil
+
+	logger.Info("Succeeded in scaling this cluster")
```
Comment (fanux) [inline apply/applytype/local.go:155]: `Scaling the cluster succeeded`

File: apply/applytype/local.go [hunk 5]
```diff
@@ -91,103 +92,135 @@ func (c *Applier) unMountClusterImage() error {
...
 
-	return nil
+	logger.Info("Succeeded in upgrading current cluster from version(%s) to version(%s)", info.GitVersion, desiredMetadata.Version)
+
+	return true, nil
 }
 
 func (c *Applier) initCluster() error {
-	logger.Info("Current cluster is nil will create new cluster")
+	logger.Info("There is NOT an existing cluster, start creating a new cluster")
```
Comment (fanux) [inline apply/applytype/local.go:197]: `Start to create a new cluster`

File: sealer/cmd/delete.go [hunk 6]
```diff
@@ -64,6 +64,10 @@ delete all:
 				return fmt.Errorf("the delete parameter needs to be set")
 			}
 			deleteClusterName, err = utils.GetDefaultClusterName()
+			if err == utils.ErrClusterNotExist {
+				fmt.Println("Find no exist cluster, skip delete")
```
Comment (fanux) [inline sealer/cmd/delete.go:68]: Using logger?

**Review by fanux — APPROVED** [review]

**Pull request merged**